### PR TITLE
Update plugin-template

### PR DESCRIPTION
**Pull request by mikedep333**
Includes increasing pulp-workers from 2 to 9

[noissue]

### CODE REVIEW

File: template_config.yml [hunk 1]
```diff
@@ -32,6 +32,7 @@ flake8: true
 flake8_ignore: []
 github_org: pulp
 issue_tracker: github
+kanban: true
 lint_requirements: true
 noissue_marker: '[noissue]'
 parallel_test_workers: 8
```
Comment (mdellweg) [inline template_config.yml:38]: Is this an old variable?

Comment (mikedep333) [reply template_config.yml:38]: It got generated when I ran the plugin-template command.

Comment (mdellweg) [reply template_config.yml:38]: Can we remove `parallel_test_workers` if we are not using it?

**Comment by dkliban** [comment]
Closing because it is out of date.

**Pull request closed without merging**